pkg/controller/pravega/pravega_controller.go

```diff
@@ -97,7 +97,7 @@ func makeControllerPodSpec(p *api.PravegaCluster) *corev1.PodSpec {
 				ReadinessProbe: &corev1.Probe{
 					Handler: corev1.Handler{
 						Exec: &corev1.ExecAction{
-							Command: util.HealthcheckCommand(9090),
+							Command: util.ReadinessHealthcheckCommand(10080),
```

Comment:
In the ReadinessProbe configuration, we seem to be setting only "periodSeconds" and "failureThreshold".... These values need to be adjusted as per the new check....

 "failureThresold" value is very high... perhaps required by the earlier check... 
but now based on the new check, it can be removed ( _so default of 3 would be used_).

Based on K8s documentation for probes:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#probe-v1-core
it may be good to set "initialDelaySeconds" to avoid doing the probe till the controller server has had time to come up successfully...
Also consider setting "timeoutSeconds" to guard from situations where curl hangs...
Also consider setting "successThreshold" ...

Reply:
yes, now I have removed "failureThresold" and "periodSeconds" as there default values are enough and have added the other 3 fields.

Reply:
Ok